cumulus/parachains/integration-tests/emulated/tests/people/people-rococo/src/tests/reap_identity.rs

```diff
@@ -0,0 +1,138 @@
+use crate::*;
+use emulated_integration_tests_common::xcm_emulator::Get;
+use pallet_identity::{legacy::IdentityInfo, Data};
+use people_rococo_runtime::people::IdentityInfo as IdentityInfoParachain;
+use rococo_runtime::MaxAdditionalFields;
+use rococo_system_emulated_network::{
+	rococo_emulated_chain::RococoRelayPallet, RococoRelay, RococoRelaySender,
+};
+
+fn identity_relay() -> IdentityInfo<MaxAdditionalFields> {
+	IdentityInfo {
+		display: Data::Raw(b"xcm-test".to_vec().try_into().unwrap()),
+		additional: Default::default(),
+		legal: Default::default(),
+		web: Default::default(),
+		riot: Default::default(),
+		email: Default::default(),
+		pgp_fingerprint: None,
+		image: Default::default(),
+		twitter: Default::default(),
+	}
+}
+
+fn identity_parachain() -> IdentityInfoParachain {
+	IdentityInfoParachain {
+		display: Data::Raw(b"xcm-test".to_vec().try_into().unwrap()),
+		legal: Data::Raw(b"The Right Ordinal Xcm Test, Esq.".to_vec().try_into().unwrap()),
+		web: Data::Raw(b"https://xcm-test.io".to_vec().try_into().unwrap()),
+		matrix: Data::Raw(b"@xcm-test:matrix.org".to_vec().try_into().unwrap()),
+		email: Data::Raw(b"xcm-test@gmail.com".to_vec().try_into().unwrap()),
+		pgp_fingerprint: None,
+		image: Data::Raw(b"xcm-test.png".to_vec().try_into().unwrap()),
+		twitter: Data::Raw(b"@xcm-test".to_vec().try_into().unwrap()),
+		github: Data::Raw(b"xcm-test".to_vec().try_into().unwrap()),
+		discord: Data::Raw(b"xcm-test#0042".to_vec().try_into().unwrap()),
+	}
+}
```

Comment:
I've updated so that the data under the matching fields can be identical. Re the new case about mutating this data on the parachain, I'll submit that in a new PR after this one is merged.